tests/zfs-tests/tests/functional/cli_root/zfs_mount/zfs_mount_007_pos.ksh

```diff
@@ -123,7 +123,8 @@ for property in ${properties[@]}; do
 
 	# Set filesystem property temporarily
 	reverse_opt=$(get_reverse_option $fs $property)
-	log_must zfs mount -o remount,$reverse_opt $fs
+	log_must zfs unmount $fs
+	log_must zfs mount -o $reverse_opt $fs
```

Comment:
On Linux at least this is a subtly different test scenario, but I agree it should still fulfill the intent of the test.